src/hyperlight_host/src/hypervisor/hyperv_linux.rs

```diff
@@ -188,6 +199,31 @@ impl HypervLinuxDriver {
         let (debug, gdb_conn) = if let Some(gdb_conn) = gdb_conn {
             let debug = MshvDebug::new();
 
+            // The bellow intercepts make the vCPU exit with the Exception Intercept exit code
+            // Install intercept for #DB exception
+            vm_fd
+                .install_intercept(mshv_install_intercept {
+                    access_type_mask: HV_INTERCEPT_ACCESS_MASK_EXECUTE,
+                    intercept_type: hv_intercept_type_HV_INTERCEPT_TYPE_EXCEPTION,
+                    // Exception handler #DB
+                    intercept_parameter: hv_intercept_parameters {
+                        exception_vector: 0x1,
```

Comment:
Shall we make these magic numbers constants or at least comment what they are and where they come from?

Reply:
I did mention above the handler for `#DB`. This is the way it is mentioned in the Intel Spec.
I'll add the specific page here also.